SU-SWS/stanford-jumpstart-deployer
# HSDO-293: Courses 4.x upgrade for jsa js+ jspvpsa jse

# Stanford Courses Upgrade
### To test:
1. Build a new site of each product JS+, JSA, JSVPSA, JSE
2. Enable stanford_courses
3. Create an importer and import some courses
4. Test to ensure values in field collections are correct
5. Profit
